.ci/generate_bear_requirements.py

```diff
@@ -168,6 +171,18 @@ def get_pip_requirements(requirements):
 def get_cabal_requirements(requirements):
     return _get_requirements(requirements, '==')
 
+def get_languages(bears):
+    language_dict = {}
+    for bear in bears:
+        language_dict[str(bear.name)] = list(sorted(bear.LANGUAGES))
+    for value in language_dict.values():
+        for n,i in enumerate(value):
+            if(i=='All'):
+                value[n]=True
```

Comment:
this is why `true` is going into the yaml .

At https://github.com/coala/coala-bears/pull/2688#discussion_r225752332 , I said that it should be `Bear: true` or `Bear: false`.  It could also be `Bear:`

It can not be
```yaml
Bear:
  - true
```